configuration/locations/locations.csv

```diff
@@ -1,73 +1,81 @@
+Sante Kominotè,dce85b2e-6946-4798-9037-d00f123df7bd,,Community Health clinic at Mirebalais Hospital (Community Health clinic).,Hôpital Universitaire de Mirebalais - Prensipal,M011,,,,TRUE,,TRUE,TRUE,,TRUE,,,,,,TRUE,,,,,,TRUE,TRUE,,,,TRUE,TRUE,,,TRUE,TRUE,,TRUE
+Swen Entansif,b6fcd85f-5995-43c9-875f-3bb2299087ff,,ICU at Mirebalais Hospital (Intensive Care Unit),Hôpital Universitaire de Mirebalais - Prensipal,M018,,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,TRUE
+Swen Entansif Neonatal,62a9500e-a1a5-4235-844f-3a8cc0765d53,,NICU at Mirebalais Hospital (Neonatal Intensive Care Unit),Hôpital Universitaire de Mirebalais - Prensipal,M014,,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,TRUE,,TRUE,,,,TRUE,TRUE,,,TRUE,,,,,TRUE,,,TRUE
+Travay e Akouchman,dcfefcb7-163b-47e5-84ae-f715cf3e0e92,,Labor and Delivery at Mirebalais Hospital (Births),Hôpital Universitaire de Mirebalais - Prensipal,M023,,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,TRUE,,TRUE,TRUE,,,TRUE,TRUE,,,TRUE,,,,,TRUE,,,TRUE
+COVID-19 | Izolman | HSN,6dcf6d61-b7b8-4e4e-bc2b-519b13f0a03a,TRUE,COVID-19 Isolation,Saint-Marc (HSN),,,,TRUE,,,TRUE,TRUE,,,,,,,,,TRUE,,,,,,,,,TRUE,TRUE,,,TRUE,TRUE,,,TRUE
+COVID-19 | UMI | HSN,387956c5-c991-4a2d-a03e-c53c11200ef7,TRUE,Inite maladi enfeksyon (UMI) COVID-19,Saint-Marc (HSN),,,,TRUE,,,TRUE,TRUE,,,,,,,,,TRUE,,,,,,,,,TRUE,TRUE,,,TRUE,TRUE,,,TRUE
+Sante Mantal | HSN,85c67863-1341-4bcc-a159-35cf202f3a25,,Sante Mantal | HSN,Saint-Marc (HSN),,,,,,,,,,TRUE,,,,,,,,,TRUE,,,,,,,TRUE,TRUE,,,,,,,
+Sante Fanm | HSN,d6346dbc-9078-45ea-a3ab-c9937e799bcc,,Women's Outpatient clinic at HSN (Sante Fanm).,Saint-Marc (HSN),,,,,TRUE,,TRUE,TRUE,,TRUE,,,,TRUE,,TRUE,TRUE,TRUE,TRUE,,,TRUE,TRUE,,,TRUE,TRUE,,,,TRUE,,,TRUE
+Klinik MNT | HSN,47521fa1-ac7f-4fdb-b167-a86fe223a3d9,,The NCD clinic at HSN.,Saint-Marc (HSN),,,,,,,TRUE,,,TRUE,,,,,,,,,,TRUE,,,,TRUE,,,TRUE,,,,,,,
+Sante Mantal | Cange,61fcc62a-84c4-4d52-a3cd-25f58ff8c303,,Sante Mantal | Cange,Cange,,,,,,,,,,TRUE,,,,,,,,,TRUE,,,,,,,TRUE,TRUE,,,,,,,
+Sante Fanm | Cange,9aab3f94-eef5-4269-83a8-97cfe0679828,,Women's Outpatient clinic at Cange (Sante Fanm).,Cange,,,,,TRUE,,TRUE,TRUE,,TRUE,,,,TRUE,,TRUE,TRUE,TRUE,TRUE,,,TRUE,TRUE,,,TRUE,TRUE,,,,TRUE,,,TRUE
+Klinik MNT | Cange,3c609c2d-a9a2-4e59-932d-be53e6a39199,,The NCD clinic at Hinche.,Cange,,,,,,,TRUE,,,TRUE,,,,,,,,,,TRUE,,,,TRUE,,,TRUE,,,,,,,
+Sante Mantal | Hinche,67b577e0-ff06-41a3-92f4-09dc27a5ae24,,Sante Mantal | Hinche,Hinche,,,,,,,,,,TRUE,,,,,,,,,TRUE,,,,,,,TRUE,TRUE,,,,,,,
+Sante Fanm | Hinche,602175ae-24d5-4711-bd28-511e4d702460,,Women's Outpatient clinic at Hinche (Sante Fanm).,Hinche,,,,,TRUE,,TRUE,TRUE,,TRUE,,,,TRUE,,TRUE,TRUE,TRUE,TRUE,,,TRUE,TRUE,,,TRUE,TRUE,,,,TRUE,,,TRUE
+Klinik MNT | Hinche,9705b785-0f00-41d1-a29b-3b357a307cdb,,The NCD clinic at Hince.,Hinche,,,,,,,TRUE,,,TRUE,,,,,,,,,,TRUE,,,,TRUE,,,TRUE,,,,,,,
```

Comment:
This diff came out pretty bad, I assume because I added a new column "Void/Retire" to every row